README.md

```diff
@@ -113,8 +113,10 @@ This library currently supports the following:
 - ES384
 
 ### RSA
-`jsonwebtoken` can only read DER encoded keys currently. If you have openssl installed,
-you can run the following commands to obtain the DER keys from PKCS#1 (ie with `BEGIN RSA PUBLIC KEY`) .pem.
+`jsonwebtoken` can read DER and PEM encoded keys.
+
+If you want to use DER encoded keys review the following.
```

Comment:
this sentence seems like it's missing stuff?

Reply:
I've updated the readme a little.